chart/values.yaml

```diff
@@ -0,0 +1,94 @@
+provisionDataStore:
+  cassandra: false
+  elasticsearch: false
+
+storage:
+  type: elasticsearch
+  elasticsearch:
+    scheme: http
+    host: elasticsearch-master
+    port: 9200
+    user: elastic
+    usePassword: false
+
+agent:   
+  resources:
+    limits:
```

Comment:
Always put the same order for request->limits... req first.

Reply:
Logique, j'ai copié des values.yaml de chaque outil et ils utilisent pas la même convention j'ai pas porté attention